### DISCUSSION TITLE
Vitotronic 200 KW6B (mit Vitocrossal 300)

### DISCUSSION THREAD

**Discussion by githubler** [openv/openv#656] [General]
Hallo,
ich möchte gerne openv (genauer einen Wemos D1 mini (ESPHome)) mit meiner Anlage nutzen. Leider finde ich nur etwas zur V200 KW1/2.
Wird die KW6B auch unterstützt? Und was muss ich hier beachten?

Über einen kurzen Hinweis würde ich mich sehr freuen!

**Comment by sbidy** [comment]
Hey,

schraube hier grade an einem ähnlichen Setup herum.
Im Prinzip ein Vitoladens 300-C (BC3 / 2-Stufig) mit einer Vitotronic 200 KW6B (nach Typenschild).

Ich hab auch das Setup mit einem ESP8266 sowie dem ESPhome (https://deploy-preview-2737--esphome.netlify.app/components/optolink) Modul umgesetzt. Soweit tut auch das meiste.

Ich habe die Adresse von einer V200KW2 verwendet.
Ein kleiner Ausschnitt aus meiner Konfig mit funktionierenden Adressen:
```
    name: Kesseltemperatur
    address: 0x0802
    byte: 2

    name: Speichertemperatur
    address: 0x0804
    byte: 2

    name: Vorlauf IST HK2
    address: 0x3900
    byte: 2

    name: Soll Raumtemperatur HK1 (read/write)
    address: 0x2306
    byte: 1

    name: Soll Raumtemperatur HK2 (read/write)
    address: 0x3306
    byte: 1

    name: Heizkreispumpe HK1
    address: 0x2906
    byte: 1

    name: Heizkreispumpe HK2
    address: 0x3906
    byte: 1

    name: Warmwasser Freigabe
    address: 0x650A
    byte: 1

```

ABER wenn ich eine Device-Info abfrage, bekomme ich die **Device ID 20C8 mit Stand 3D** und nicht wie erwartet die für die ein V200xx.
Die 20C8 wäre ein VPlusHO1, was aber nicht ganz stimmen kann ??

UND eine der für mich wichtigsten Parameter, die Brenne-Stufen-Freigabe, funktioniert nicht richtig. Ich bekomme bei Stufe 1 mit Adresse `0x0842` (V200KW2) gar keinen anständigen Wert und mit der vom VPlusHO1 `0x55DE` was halb plausibles (ändert sich zumindest von 00 auf c1 bei Stufe 1 Start). Für die zweite Stufe passen keine der Werte vom VPlus oder V200.

Ich habe mich auch schon durch die XMLs und andere Datepunktlisten gewühlt, aber irgendwie passt nichts sehr zuverlässig auf diese Steuerung.

Hat jemand einen Tipp bzw. einen funktionierenden Wert für die Brenner? Ich denke, das wäre noch der einzige Wert, der für mich wichtig wäre.

**Comment by sbidy** [comment]
Mir scheint, dass die Steuerung die zweite Stufe nicht kennt bzw. nicht zur Verfügung stellt, was echt schräg wäre.
@githubler hast du einen funktionierenden Wert für die zweite Brennerstufe?

Stufe 1 geht auch mit:
```
    name: Brenner Stufe 1 (h)
    address: 0x0886
    bytes: 4
    div_ratio: 3600
```

Die Adresse ist wiederum für den VPlusHO1 ....

**Comment by sbidy** [comment]
Ok der Datenpunkt `0x0886` gibt zwar ein Wert zurück, jedoch nicht die Brennerstunden (-sekunden)....

**Comment by githubler** [comment]
Hi @sbidy,

sorry bin leider noch nicht so weit. 😔
Ich befürchte ich hab meine IRs zerschossen.. Zumindest bekomme ich keine Werte aus meinem Aufbau.
Ich hoffe ich komme wieder dazu mich darum zu kümmern, leider ist die Zeit derzeit bei mir zu knapp bemessen. 

Aber schon mal Danke für deine Infos! 
Somit scheint die KW6B (im Prinzip) zu funktionieren. 🥳
